Directory.Build.props

```diff
@@ -3,6 +3,14 @@
     <PackageOutputPath>$(MSBuildThisFileDirectory)outputpackages\</PackageOutputPath>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <CodeAnalysisRuleSet>$(SolutionDir)BotBuilder-DotNet.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="StyleCop.Analyzers" Version="1.0.2" PrivateAssets="all" />
```

Comment:
I don't understand why this is both here, and in each of the .csproj files. Could you elaborate? 

It seems like only 1 location would be needed.

Reply:
Hi @cleemullins, we update the PR for removing the double reference. Now, the _StyleCop.Analyzers_ NuGet is only referenced in the _Directory.Build.props_ file.